gerrit/install.sls

```diff
@@ -66,12 +66,12 @@ install_{{ name }}_plugin:
 {% endfor %}
 
 gerrit_war:
-  cmd.run:
-    - name: wget -qO {{ settings.base_directory }}/{{ gerrit_war_file }} {{ settings.package.base_url }}/{{ gerrit_war_file }}
-    - cwd: {{ settings.base_directory }}
+  file.managed:
```

Comment:
This doesn't seem to take into account operating systems outside of FreeBSD and it needs to. This is especially true when you call the file freebsd-rc. This should probably be grain based like the section below.